### PR TITLE
Fix GitHub ci

### PR DESCRIPTION
**Pull request by area363**
- https://github.com/planetarium/lib9c/commit/90f8dc45a9fc5df09c341a16169cc1e0a6b8c977 fixes https://github.com/planetarium/lib9c/actions/runs/5996957418/job/16262426753#step:4:16.

- https://github.com/planetarium/lib9c/commit/4b526fc3278a37e694384f23e91d2e9f38e77f6c fixes https://github.com/planetarium/lib9c/actions/runs/5958008686/job/16161621054#step:5:42.

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: release
 on:
   push:
     tags:
-    - v*
+    - "*"
```
Comment (moreal) [inline .github/workflows/release.yaml:6]: When I saw the GitHub Actions logs by your description, this part seems not related 👀

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -16,15 +16,17 @@ jobs:
     - uses: actions/setup-dotnet@v3
       with:
         dotnet-version: 6.0.100
-    - name: bulid
+    - name: build
```
Comment (moreal) [inline .github/workflows/publish.yml:19]: 👍🏻

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -16,15 +16,17 @@ jobs:
...
         if [[ ! "$GITHUB_REF" =~ ^refs/tags/* ]]; then
           project_suffix=dev.${{ github.sha }}
           dotnet_args="$dotnet_args --version-suffix $project_suffix"
         fi
-        dotnet build "$dotnet_args"
-        dotnet pack "$dotnet_args"
+        # shellcheck disable=SC2086
+        dotnet build $dotnet_args
+        # shellcheck disable=SC2086
+        dotnet pack $dotnet_args
```
Comment (moreal) [inline .github/workflows/publish.yml:29]: In the shellcheck wiki, it suggests using like `${dotnet_args}` 🤔 But the current version can be merged too! 👍🏻 

https://github.com/planetarium/lib9c/issues/2103

File: .github/workflows/release.yaml (hunk 1 above)
Comment (area363) [reply .github/workflows/release.yaml:6]: Oops, I pushed this change by mistake. Thanks for checking.

**Review by moreal — APPROVED** [review]
